src/components/EyesOnMe.js

```diff
@@ -1 +1,19 @@
-// Code EyesOnMe Component Here
+import React from "react"
+
+const EyesOnMe = () => {
+  const handleFocus = () => {
+    console.log("Good!")
+  }
+
+  const handleBlur = () => {
+    console.log("Hey! Eyes on me!")
+  }
+
+  return (
+    <>
+      <button onFocus={handleFocus} onBlur={handleBlur}>Eyes on me</button>
+    </>
```

Comment:
No need for the fragment here. 
```suggestion
      <button onFocus={handleFocus} onBlur={handleBlur}>Eyes on me</button>
```